polyfills/Window.location.origin/polyfill.js

```diff
@@ -0,0 +1,7 @@
+var origin = window.location.protocol + '//' + window.location.hostname + (window.location.port ? ':' + window.location.port : '');
```

Comment:
This value will leak into all other polyfills.  We don't put a closure around each polyfill since many have their own anyway.  In this case you don't really need one, you could just move the expression inline on line 5.